src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -685,25 +704,26 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
         return t;
     }
 
-    private void verifyParameterPosition(int param) throws SQLServerException {
+    private Map<String, Object> getParameterInfo(int param) throws SQLServerException {
         boolean bFound = false;
-        try {
-            if (((SQLServerPreparedStatement) stmtParent).bReturnValueSyntax && isTVP) {
-                bFound = rsProcedureMeta.absolute(param);
-            } else {
-                bFound = rsProcedureMeta.absolute(param + 1); // Note row 1 is the 'return value' meta data
+        if ((stmtParent).bReturnValueSyntax && isTVP) {
```

Comment:
I can't figure out why the variable would be named `bFound`. Please rename it to a descriptive one.